src/Storages/MergeTree/registerStorageMergeTree.cpp

```diff
@@ -768,6 +771,7 @@ static StoragePtr create(const StorageFactory::Arguments & args)
             std::move(storage_settings),
```

Comment:
Why do you need a separate field for storing zookeeper_cluster when `storage_settings` is available?

Reply:
It's not necessary, should be removed.